core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -2483,6 +2483,20 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     }
   }
 
+
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk", "kraft"))
+  def testCreateAndCloseConsumerWithNoAccess(quorum: String): Unit = {
+    val consumer = createConsumer()
+    try {
+      // Close consumer without consuming anything. close() call should pass successfully and throw no exception.
+      consumer.close()
+    } catch {
+      case e: Throwable =>
+        fail(s"Exception not expected on closing consumer: $e")
+    }
```

Comment:
Could we use `assertDoesNotThrow` instead? I.e.:
```java
assertDoesNotThrow(() => consumer.close(), s"Exception not expected on closing consumer)
```

Reply:
@showuon, sounds good. Updated to use assertDoesNotThrow. 

Note: I cannot pass "() => consumer.close()" directly to assertDoesNotThrow because it causes compile error:
```
ambiguous reference to overloaded definition,
both method assertDoesNotThrow in class Assertions of type [T](x$1: org.junit.jupiter.api.function.ThrowingSupplier[T], x$2: String): T
and  method assertDoesNotThrow in class Assertions of type (x$1: org.junit.jupiter.api.function.Executable, x$2: String): Unit
match argument types (() => Unit,String)
```